### PR TITLE
add: contact us form component

### PR DESCRIPTION
**Pull request by GustavoUT22**
I just added the "Contact Us" form component. Take a look and make sure to run 'npm install' to have all the necessary dependencies installed.

### CODE REVIEW

File: src/components/FormContactUs/Form.styles.ts
[the remark this answers is not in the document]
Comment (GustavoUT22) [reply src/components/FormContactUs/Form.styles.ts:1]: solved

File: src/components/FormContactUs/Form.styles.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components'
+
+export const FormWrapper = styled.form`
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+`
+export const Error = styled.span`
+  font-size: 0.75rem;
+  color: red;
```
[the remark this answers is not in the document]
Comment (GustavoUT22) [reply src/components/FormContactUs/Form.styles.ts:10]: solved

File: src/components/FormContactUs/Form.styles.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  margin-bottom: 0.25rem;
+`
+export const TitleForm = styled.h1`
```
[the remark this answers is not in the document]
Comment (GustavoUT22) [reply src/components/FormContactUs/Form.styles.ts:13]: solved

File: src/components/FormContactUs/Form.styles.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  text-align: center;
+  margin: 1rem 0rem;
+`
+
+export const ContainerForm = styled.div`
+  padding: 0.75rem;
+  background-color: #eceaea;
```
[the remark this answers is not in the document]
Comment (GustavoUT22) [reply src/components/FormContactUs/Form.styles.ts:20]: solved

File: src/components/FormContactUs/Form.tsx
[the remark this answers is not in the document]
Comment (GustavoUT22) [reply src/components/FormContactUs/Form.tsx:1]: solved

File: src/components/FormContactUs/Form.yup.ts
[the remark this answers is not in the document]
Comment (GustavoUT22) [reply src/components/FormContactUs/Form.yup.ts:1]: solved

**Review by CrhistianT7 — APPROVED** [review]
Good job!!

**Pull request merged**